### PR TITLE
Feature/contact social media profiles

### PR DESCRIPTION
**Pull request by janmassive**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no

#### What's in this PR?

Ability to add social media profile to contacts and accounts.

### CODE REVIEW

**Comment by janmassive** [comment]
@danrot

File: src/Sulu/Bundle/ContactBundle/Entity/SocialMediaProfileType.php [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+/**
+ * Type of social media profile (i.e. Facebook,...).
+ *
+ * @ExclusionPolicy("All")
+ */
+class SocialMediaProfileType implements \JsonSerializable
+{
+    const TYPE_FACEBOOK = 'social_media_profile.facebook';
+    const TYPE_TWITTER = 'social_media_profile.twitter';
+    const TYPE_INSTAGRAM = 'social_media_profile.instagram';
```
Comment (danrot) [inline src/Sulu/Bundle/ContactBundle/Entity/SocialMediaProfileType.php]: This way is quite hard coded, and not extensible at all 😕 Don't you want to put that into the configuration?

File: src/Sulu/Bundle/ContactBundle/Resources/translations/sulu/backend.en.xlf [hunk 2]
```diff
@@ -642,6 +642,22 @@
                 <source>labels.success.user.activated</source>
                 <target>User enable</target>
             </trans-unit>
+            <trans-unit id="235b14f26g712345e44gs2e9335a8d63" resname="public.socialMediaProfile">
+                <source>public.socialMediaProfile</source>
+                <target>Social media profile</target>
+            </trans-unit>
+            <trans-unit id="235b14f26g712345e44gs2e9335a8d73" resname="social_media_profile.facebook">
+                <source>social_media_profile.facebook</source>
```
Comment (danrot) [inline src/Sulu/Bundle/ContactBundle/Resources/translations/sulu/backend.en.xlf]: I think I would replace all of these translations with a simple string... The names never change, and it makes adding a new one harder... But let me discuss that with @chirimoya first.

File: UPGRADE.md [hunk 3]
```diff
@@ -2,6 +2,15 @@
 
 ## dev-develop
 
+### Social media profile fixtures
+
+Add fixtures for social media profile of contacts. Run following command to
+add the fixtures:
+
+```bash
+INSERT INTO co_social_media_profile_types (id, name) VALUES ('1', 'social_media_profile.facebook'), ('2', 'social_media_profile.twitter'), ('3', 'social_media_profile.instagram');
```
Comment (danrot) [inline UPGRADE.md]: You should also fix that 😃

**Pull request merged**